packages/discord-bot/src/handlers/forward.ts

```diff
@@ -34,6 +35,18 @@ export const forwardHandler: CommandHandler = async (
     return;
   }
 
+  // pass ID of parent channel if command used in a thread.
+  if (
+    (await praiseAllowedInChannel(
```

Comment:
Consider the order of the checks. Checkling that user has permissions to run command should probably happen before we check if channel allows praise.